Storage/Site.swift

```diff
@@ -85,4 +85,11 @@ public class Site: Identifiable {
         self.title = title
         self.bookmarked = bookmarked
     }
+
+    // Helper property to quickly get the favicon url
+    public var faviconURL: NSURL? {
+        return icon?.url != nil ? NSURL(string: (icon?.url)!) : nil
```

Comment:
Similar comment for the `tileURL` property (even though its not part of this PR). Looking at the way a `SuggestedSite` and regular `Site` is created, I don't think we need this property and we can simply just use the existing `url` property. Ideally we keep this object pretty slim because all of it's properties get populated from a database query and it might be confusing to conflate non-DB filled values with others.